weather_api/config.py

```diff
@@ -9,7 +9,7 @@ class Settings(BaseSettings):
     api_root_path: str = "/"
 
     met_api_url: str = "https://api.met.no/weatherapi"
-    api_description: str = "This is a RESTful service that provides accurate and up-to-date weather information based on data sourced from api.met.no."
+    api_description: str = 'This is a RESTful service that provides accurate and up-to-date weather information based on data sourced from <a href="https://api.met.no">https://api.met.no</a>. <br/>The data are freely available for use under a <a href="https://api.met.no/doc/License">Creative Commons license</a>.'
```

Comment:
Will we ever need to be able to configure the description? Why not just declare this in `__main__.py`?

Reply:
No, probably not. But wanted to keep the __main__.py free from clutter, so put it in the settings.